docs/source/visint.rst

```diff
@@ -1,6 +1,33 @@
 Visualization and Interaction
 *****************************
 
+Plotting a 1D Sweep
+==================
+
+A function called ``plot_sweep1d`` has been created to facilitate creating, solving, and plotting the results of a single-variable sweep. Example usage is as follows:
+
+.. literalinclude:: examples/plot_sweep1d.py
+
+The result looks like:
+
+.. code-block:: text
+
+    Solving over 20 passes.
+    Sweeping took 0.34 seconds.
+
+.. figure:: examples/plot_sweep1d.png
+    :align: center
+
+.. code-block:: text
+
+    Solved after 7 passes.
+    Possible log error +/-0.000476
+    Autosweeping took 0.11 seconds.
+
+.. figure:: examples/plot_autosweep1d.png
+    :align: center
```

Comment:
Sounds good. This might increase testing time if it means we have to install matplotlib on the testing servers.